opencensus/proto/stats/metrics/metrics.proto

```diff
@@ -232,35 +233,8 @@ message DistributionValue {
     // The observation (sampling) time of the above value.
     google.protobuf.Timestamp timestamp = 2;
 
-    // The context of a span to be attached to Exemplar.
-    message SpanContext {
-
-      // A unique identifier for a trace showing this exemplar.
-      // It is a 16-byte array.
-      bytes trace_id = 1;
-
-      // A unique identifier for a span showing this exemplar.
-      // It is an 8-byte array.
-      bytes span_id = 2;
-    }
-
-    // A set of (label, value) pairs which were dropped during aggregation,
-    // attached to Exemplar.
-    message DroppedLabels {
-
-      // A set of (label, value) pairs.
-      map<string, string> label = 1;
-    }
-
     // Contextual information about the example value.
-    oneof attachment {
-
-      // A span context attachment.
-      SpanContext span_context = 3;
-
-      // A dropped labels attachment.
-      DroppedLabels dropped_labels = 4;
-    }
+    repeated google.protobuf.Any attachments = 3;
```

Comment:
Why does this field use Any, instead of a string to string map, as in https://github.com/census-instrumentation/opencensus-specs/pull/122?

Reply:
Done. Cloud Monitoring uses Any as Exemplar attachment in their proto definition, though it makes our life easier to use a map on OpenCensus side.